src/main/java/com/duke/Duke.java

```diff
@@ -0,0 +1,197 @@
+package com.duke;
+
+import com.file.FileManager;
+import com.task.Deadline;
+import com.task.Event;
+import com.task.Task;
+import com.task.Todo;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.io.IOException;
+
+public class Duke {
+
+    public static FileManager fileTask;
+    static ArrayList<Task> list = new ArrayList<>();
+    public static String[] dukeLines = new String[100];
+    public static int dukeLineCount = 0;
+    public final static String LINESEPARATER =  "\t______________________________________________________________________";
+
+    public static void addTask(Task item) {
+        list.add(item);
+    }
+    private boolean active = false;
+
+    public Duke(){
+        active = true;
+        printLine();
+        System.out.println("\tHello from");
+        System.out.println("\t ____        _        ");
+        System.out.println("\t| _ \\ _   _| | _____ ");
+        System.out.println("\t| | | | | | | |/ / _ \\");
+        System.out.println("\t| |_| | |_| |   <  __/");
+        System.out.println("\t|____/ \\__,_|_|\\_\\___|");
+        loadFromFile();
+        endLine();
+    }
+
+
+    public void loadFromFile(){
```

Comment:
How about adding a method description as [javadocs](https://se-education.org/guides/conventions/java/basic.html#comments) here? `loadFromFile()` is doing a couple of things and it would readers to know quickly what those things are.